ang/civicase/ActivitiesCalendar.js

```diff
@@ -614,16 +608,17 @@
     }
 
     /**
-     * Creates the url for the "see all" link, based on the given date
+     * Creates the url for the "see all" link, based on the base URL provided to
+     * the controller and the given date passed as a parameter.
      * (the link will send the user to the activity feed, filtered by that date)
      *
      * @return {Date} date
      * @return {TrustedValueHolderType}
      */
     function seeAllLinkUrl (date) {
-      var urlParams = getSeeMoreQueryParams(date);
+      var urlParams = getSeeMoreActivityFilters(date);
 
-      return $sce.trustAsResourceUrl('#/case?' + $.param(urlParams));
+      return getActivityFeedUrl({ filters: urlParams });
```

Comment:
The `getActivityFeedUrl` already supports having different params for filtering, why do we need a new `filters` object?

Reply:
Changed the name of the var to `activityFilters` the difference between this var and the other params is that this one is going to be passed to the `af` url param and the others are passed directly as url params. Like `caseId` for example.